### PR TITLE
picocom: update to 2.0

### PR DESCRIPTION
**Pull request by allan**

### CODE REVIEW

**Comment by chneukirchen** [comment]
Using `sed -i` is fine for templates.

**Comment by allan** [comment]
and `ed` is fine for inplace editing everywhere:)

**Comment by chneukirchen** [comment]
Is it in base-chroot?

**Comment by pullmoll** [comment]
It ain't, thus the `hostmakedepends=ed`. I'd prefer to see `sed -i` as well.. the `ed` syntax is..umm..awkward.

**Comment by dominikh** [comment]
We use sed in all our other templates, please use sed :)

**Comment by allan** [comment]
There is some ed in other templates like e0b08294e94465711372b2aeb10c3ba72356af8c.  I vote for ed in base-chroot!!

**Comment by Gottox** [comment]
I vote for a real patch ;)

**Comment by Gottox** [comment]
... and no, ed shouldn't be in base. There's absolutely no reason for that.

**Comment by xtraeme** [comment]
Please create a real patch and submit it upstream, there's no need for `ed(1)` nor `sed(1)`.

Thanks

File: srcpkgs/picocom/template [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+version=2.0
 revision=1
 short_desc="Minimal dumb-terminal emulation program like minicom"
 maintainer="allan <mail@may.mooo.com>"
 license="GPL-2"
 homepage="https://github.com/npat-efault/picocom"
 distfiles="https://github.com/npat-efault/picocom/archive/${version}.tar.gz"
-checksum=6a3e6ba74eec3dd111d5c6f8245b8f08ee54db783e5d319a8589e8f7a4a108f0
+checksum=e0f7095c118f43c1c09787a16ef5c7d4c4ad4457dd3b1281efcc4cdb8906af9a
+hostmakedepends=ed
```
Comment (xtraeme) [inline srcpkgs/picocom/template]: `hostmakedepends` is not necessary anymore.

**Pull request merged**